imports/api/validators/server/methods.js

```diff
@@ -26,7 +26,7 @@ Meteor.methods({
     // async 'Validators.getAllDelegations'(address){
     'Validators.getAllDelegations'(address){
         this.unblock();
-        let url = API + '/cosmos/staking/v1beta1/validators/'+address+'/delegations';
+        let url = API + '/cosmos/staking/v1beta1/validators/' + address +'/delegations?pagination.limit=500';
```

Comment:
Make this query by page number.

Reply:
I have added the changes that we talked about today in the office. As grpc is currently very slow when querying the list of delegations, I have replaced the delegation list displayed under the validator page with the total number of delegations for each validator.